lib/tasks/taxonomy/register_email_signup_page.rake

```diff
@@ -0,0 +1,46 @@
+namespace :taxonomy do
+  desc <<-DESC
+    Register a content item representing the email-alert-frontend endpoint that
+    handles taxonomy email alert subscriptions
+  DESC
+  task register_email_signup_page: :environment do
+    def base_path
+      '/taxonomy-email-signup'
+    end
+
+    def routes
+      [
+        { path: base_path, type: "exact" },
+        { path: "#{base_path}/confirm", type: "exact" },
+      ]
+    end
+
+    # The base_path and routes specified here are the only important values -
+    # we want to ensure the specified path is reserved correctly via the
+    # publishing-api. All other fields are populated with nominal values,
+    # particularly the details hash which needs to satisfy the schema
+    # definition for an email_alert_signup.
+    email_signup_page = {
```

Comment:
Can you extract this payload and test it against the schema? Otherwise it might get out of date.